modules/ROOT/pages/apicat-create-descriptor-file-cli.adoc

```diff
@@ -67,6 +67,36 @@ The presence and content of the designated descriptor file affects the result of
 
 * Existing file with content: The command fails and shows an error message that says the descriptor file already exists.
 
+=== Add Images through a Descriptor File
+
+You can add images to your APIs by adding a markdown file in the descriptor with a link to an image (.png, .jpg). You can visualize the image in the documentation page created for the Exchange Asset.
+
+Example output that is written to the descriptor file:
```

Comment:
No need for `output written` since this is not autogenerated by the create descriptor command.
`Example descriptor file` is maybe better